stable/pgadmin/templates/pvc.yaml

```diff
@@ -1,19 +1,26 @@
-{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
```

Comment:
See https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md for info on persistence

Reply:
Not really sure what I should be looking at, could you perhaps be more specific?